.cbmc-batch/source/utils.c

```diff
@@ -38,7 +38,7 @@ void assert_all_zeroes(const uint8_t *const a, const size_t len) {
 }
 
 void assert_byte_from_buffer_matches(const uint8_t *const buffer, const struct store_byte_from_buffer *const b) {
-    if (buffer) {
+    if (buffer && b) {
```

Comment:
when would we ever not have a b?

Reply:
When one tries to save a byte from a NULL buffer using `save_byte_from_array`